src/templates/template-text-classification/README.md

```diff
@@ -24,26 +24,20 @@ pip install -r requirements.txt --progress-bar off -U
 - Execute on master node
 
 ```sh
-python -m torch.distributed.launch \
+torchrun main.py \
   --nproc_per_node #:::= it.nproc_per_node :::# \
   --nnodes #:::= it.nnodes :::# \
   --node_rank 0 \
-  --master_addr #:::= it.master_addr :::# \
-  --master_port #:::= it.master_port :::# \
-  --use_env main.py \
```

Comment:
We should keep `main.py` as executable script otherwise it does not make any sense